### PR TITLE
Refactored the File Organization correctly 

### LINKED ISSUE #52: File Organization should be refactored correctly
**Actual Behaviour**

* In the current state of the application, the classes are not correctly organized into proper subpackages and packages as they should be, following the neurolab-desktop repo's organization. 
* For instance, currently, MainActivity (the Launcher class equivalent in neuro-desktop), MathBasics, NeuroSettings (the BCISettings equivalent in neuro-desktop) and other related classes are not present inside the main subpackage. 
* Also, the FeedbackSettings class is not present in the model subpackage.

**Expected Behaviour**

* All the classes should be correctly placed in the required subpackage/package. Moreover, the behavior should be refactored so that the external changes don't alter the internal structure of the application. 
* Explicitly, in accordance with the present state of our application, MainActivity, MathBasics, NeuroSettings, and other related classes should be placed in the main subpackage. FeedbackSettings should be placed in the model subpackage (and not in the settings subpackage). 
* Also, all the future classes should be placed appropriately following the neurolab-desktop's standards (which is the software system we are porting). This would help in better code organization in the long-run.

**Would you like to work on the issue?**

I would like to work on this issue.

### PR DESCRIPTION
**Pull request by suresh-gandhi**
Fixes #52 

**Changes**: 
* Placed all the classes in correct subpackage/package. Moreover, the behavior is refactored taking into account that the external changes don't alter the internal structure of the application.
* The changes are taken following the neurolab-desktop's standards (which is the software system we are porting) and for helping in better code organization in the long-run.

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

### CODE REVIEW

**Comment by suresh-gandhi** [comment]
@mariobehling @CloudyPadmal @edenyay @iamareebjamal  Could you please review the PR?

**Review by CloudyPadmal — APPROVED** [review]

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]
Changes are all good. But there are conflicts to resolve first

**Comment by suresh-gandhi** [comment]
@CloudyPadmal I have resolved the conflicts. Could you please review the PR. Conflicts came due to the lag of my branch behind the development branch(because of subsequent PR's). Rebasing solved the issue. I have also squashed them into a single commit. Just let me know if anything else needs to be changed.

File: app/src/main/java/io/neurolab/main/NeuroSettings.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package io.neurolab;
+package io.neurolab.main;
```
Comment (iamareebjamal) [inline app/src/main/java/io/neurolab/main/NeuroSettings.java]: Shouldn't this be in settings?

File: app/src/main/java/io/neurolab/model/FeedbackSettings.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package io.neurolab.settings;
+package io.neurolab.model;
```
Comment (iamareebjamal) [inline app/src/main/java/io/neurolab/model/FeedbackSettings.java]: Shouldn't this be in settings?

**Comment by suresh-gandhi** [comment]
@CloudyPadmal resolved the conflicts. Could you please review and merge. Let me know if something else needs to be changed.

**Review by mariobehling — APPROVED** [review]

**Pull request merged**